sdk/template/Azure.Template/src/properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+// Replace with test project/test project public key and uncomment to make internal members visible to
+// your test project. If not needed, this can be deleted.
+// [assembly: InternalsVisibleTo("Azure.Template.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
```

Comment:
I generally don't like folks using internals visible to. Is there a reason we think we should add it to the template?

Reply:
It is just a convenience in case someone needs it.

Reply:
OK I'm not going to try and block this change but I just was curious if there was a common pattern that is emerging that requires most people to need it.

Reply:
I think most libraries end up using it, so that is why I thought it would be nice to include.